woocommerce/payment-gateway/class-sv-wc-payment-gateway-plugin.php

```diff
@@ -181,8 +181,8 @@ protected function init_rest_api_handler() {
 	 */
 	protected function init_blocks_handler() : void {
 
-		require_once( $this->get_framework_path() . '/Blocks/Blocks_Handler.php' );
-		require_once( $this->get_framework_path() . '/payment-gateway/Blocks/Gateway_Blocks_Handler.php' );
+		require_once( $this->get_payment_gateway_framework_path() . '/Blocks/Blocks_Handler.php' );
+		require_once( $this->get_payment_gateway_framework_path() . '/payment-gateway/Blocks/Gateway_Blocks_Handler.php' );
```

Comment:
```suggestion
		require_once( $this->get_payment_gateway_framework_path() . '/Blocks/Gateway_Blocks_Handler.php' );
```
Same as above, no need to repeat `/payment-gateway`